lxd/instance/drivers/driver_qemu.go

```diff
@@ -3097,6 +3097,17 @@ func (d *qemu) deviceBootPriorities(base int) (map[string]int, error) {
 	return sortedDevs, nil
 }
 
+// shortenedSocketPath creates a shorter alternative path to a socket by using the file descriptor to the directory of the socket file.
+// Used to handle paths > 108 chars.
+func (d *qemu) shortenedSocketPath(originalSockPath string, fdFiles *[]*os.File) (string, error) {
+	configSockDir, err := os.Open(filepath.Dir(originalSockPath))
```

Comment:
Would it be worth adding a comment to the function saying that the opened file added to `fdFiles` has to be closed outside of this function if it isn't used anymore?

Reply:
It may not be necessary because `fdFiles` is used to make files visible to another process so it should be clear that the file should be closed after calling said process. But I am happy either way, just please confirm if you think I should add it.